src/hffs/fs.py

```diff
@@ -234,14 +259,13 @@ def cp_file(self, path1, path2, **kwargs):
         path1 = self._strip_protocol(path1)
         path2 = self._strip_protocol(path2)
 
+        # TODO: Wait for https://github.com/huggingface/huggingface_hub/issues/1083 to be resolved to simplify this logic
         if self.info(path1)["lfs"] is not None:
-            # Perhaps we can add a CopyOperation to the commit API in huggingace_hub?
-            headers = huggingface_hub.utils.build_hf_headers(
-                use_auth_token=self.token, library_name="hffs", library_version=__version__
-            )
+            self._api._build_hf_headers(is_write_action=True)
```

Comment:
```suggestion
            headers = self._api._build_hf_headers(is_write_action=True)
```

I think something here went wrong when merging my PR into this one. The idea is to compute the headers with `self._api` here and reuse it just after. 

(update: maybe related to `ruff`. I already experienced something being deleting without me noticing)